### DISCUSSION TITLE
High Contrast mode not working for Applications

### DISCUSSION THREAD

**Discussion by ShrutiJaiswal1494** [AvaloniaUI/Avalonia#6296] [Q&A]
Hi,

I have `High Contrast` (Settings->Ease of Access->Hight Contrast) mode enabled with a `High Contrast Black` Theme selected on my computer. When I run my Avalonia Application it still shows me the same colors (attached a picture below). Seems like it is not changing based on the high contrast theme that I have selected. However, the WPF applications are changing colors by default based on the theme selected.
 Is there something I need to add to my application to make sure Avalonia responds to the high contrast mode? Any help is much appreciated. Thanks.

![image](https://user-images.githubusercontent.com/79670853/126826835-54c28e62-e404-4ce5-a0b0-587ac29187ce.png)

**Comment by maxkatz6** [comment]
Avalonia does not support it right now. 
You need to handle system event and change the colors manually.

**Reply by ShrutiJaiswal1494** [reply]
@maxkatz6 I see. Thanks for the reply. How can I listen to SystemEvents in the Avalonia application? If you can point me to any sample code or example on how to achieve this that would be great.

**Reply by maxkatz6** [reply]
> How can I listen to SystemEvents

It's not related to avalonia. Search for target OS APIs.
If you will need window handle for anything, you can get it from Window.PlatformImpl.Handle.
